pkg/services/dashboards/database/database.go

```diff
@@ -330,19 +330,6 @@ func getExistingDashboardByIDOrUIDForUpdate(sess *db.Session, dash *dashboards.D
 		}
 	}
 
-	if dash.FolderID > 0 {
-		var existingFolder dashboards.Dashboard
-		folderExists, err := sess.Where("org_id=? AND id=? AND is_folder=?", dash.OrgID, dash.FolderID,
-			dialect.BooleanStr(true)).Get(&existingFolder)
-		if err != nil {
-			return false, fmt.Errorf("SQL query for folder failed: %w", err)
-		}
-
-		if !folderExists {
-			return false, dashboards.ErrDashboardFolderNotFound
-		}
-	}
```

Comment:
This logic is now carried out [here for dashboards](https://github.com/grafana/grafana/pull/78075/files#diff-56c23b161208f11acd3ddd098e59329339a548161430a3d7303a1e80e5ed2c8fR134-R148) and [here for folders](https://github.com/grafana/grafana/pull/78075/files#diff-62d73bdc0f7fc91a4f8dab955eba4b288815692725c72b81c1f36b87c5f123c0R864-R866).